testing/request-bank.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import requestPromise from 'request-promise';
+
+/**
+ * A server side temporary request storage which is useful for testing
+ * browser sent HTTP requests.
+ */
+export class RequestBank {
+  /**
+   * @param {string} homeUrl The URL of the dev server.
```

Comment:
how about default this value to `http://localhost:8000`?

Reply:
SG